RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -132,6 +143,18 @@ class SSHExecutionService extends RemoteExecutionService {
                 c.startSession()
                 t2 = System.nanoTime()
                 logger.postSometimesInfo(RoddyIOHelperMethods.printTimingInfo("start ssh client session", t1, t2))
+            } catch (UserAuthException ex) {
+                logger.severe(
+                        [
+                                "Could not setup SSH access with your configuration:",
+                                "- Is the user and password combination right?",
+                                "- Did you store the password and is it right?",
+                                "- Did you use keyfile authentification and are there any keyfiles available and accessible?",
```

Comment:
Just out of curiosity: What is meant by "any" keyfiles here? It will take id_{rsa,dsa}, or not?

Reply:
Think so yes.

Reply:
Yes, see [here](https://github.com/hierynomus/sshj/blob/v0.17.2/src/main/java/net/schmizz/sshj/SSHClient.java#L327).